account/src/main/java/com/user/account/services/AccountServiceClass.java

```diff
@@ -1,26 +1,27 @@
 package com.user.account.services;
 
+//import java.util.ArrayList;
+
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
-
 import com.user.account.entity.User;
-import com.user.account.repo.RepoLayer;
-
+import com.user.account.repositories.UserRepository;
 @Service
-public class AccountServiceClass implements AccountServices {
+public class AccountServiceClass implements AccountService {
 
 	@Autowired
-	private RepoLayer repoLayerVari;
+	private UserRepository userRepository;
+	
 	
 	@Override
 	public User updateUser(User user) {
-		return  repoLayerVari.save(user);
+		return  userRepository.save(user);
 	}
 
 	@Override
-	public User getUser(long accountNumber) {
-		// TODO Auto-generated method stub
-		return repoLayerVari.getOne(accountNumber);
+	public User getUser(Long accountNumber) {
+		return userRepository.getOne(accountNumber);
```

Comment:
If `accountNumber ` isn't present, what will happen?

Reply:
It will through EntityNotFoundException that is handled in catch block. User will get a message of invalid account number

Reply:
Through? Or Throw?

I don't think it's fine, EntityNotFound can happen in many places later on This approach is a bug prone in my opinion, you should throw a custom exception